hayeon/programmers-2/위장.js

```diff
@@ -0,0 +1,21 @@
+function solution(clothes) {
+  let answer = 1;
+  const clothesType = {};
+
+  for (let i = 0; i < clothes.length; i++) {
+    clothesType[clothes[i][1]] = (clothesType[clothes[i][1]] || 1) + 1;
+  }
+
+  for (let type in clothesType) {
+    answer *= clothesType[type];
+  }
+
+  return answer - 1;
+}
```

Comment:
```suggestion
function solution(clothes) {
  const clothesType = {};
  let answer = 1;

  for (let i = 0; i < clothes.length; i++) {
    clothesType[clothes[i][1]] = (clothesType[clothes[i][1]] || 1) + 1;
  }

  for (let type in clothesType) {
    answer *= clothesType[type];
  }

  return answer - 1;
}
```
- 풀이잘봤습니다!
- 코딩할 때 개발자들간의 암묵적인 약속으로 상수가 최상단으로 올려주는 컨벤션이 있습니다!
- 자스에서는 const가 상수를 의미하므로 let보다 상단에 위치시켜주세요~!

Reply:
- 헉 그렇군요!! 명심하고 다음부턴 상수를 최상단으로 올리도록 하겠습니당!! 확실히 뭔가 보기에도 안정적이긴 하네요,, 😮